pkg/commatrix/commatrix.go

```diff
@@ -0,0 +1,230 @@
+package commatrix
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	log "github.com/sirupsen/logrus"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+	"k8s.io/kubectl/pkg/util/templates"
+
+	"github.com/openshift-kni/commatrix/pkg/client"
+	commatrixcreator "github.com/openshift-kni/commatrix/pkg/commatrix-creator"
+	"github.com/openshift-kni/commatrix/pkg/endpointslices"
+	"github.com/openshift-kni/commatrix/pkg/types"
+	"github.com/openshift-kni/commatrix/pkg/utils"
+)
+
+var (
+	commatrixLong = templates.LongDesc(`
+			  Generate the communication matrix, an up-to-date communication flows matrix for all ingress flows 
```

Comment:
Generate an up-to-date communication flows matrix for all ingress flows of OpenShift (multi-node and single-node deployments) and Operators.